organize/models.py

```diff
@@ -119,9 +119,19 @@ def deploy(self):
 
         self.change_status_to(DEPLOYED)
 
-        if Event.objects.filter(city=self.city,
-                                country=self.get_country_display()).exists():
-            event = copy_event(self.previous_event, self.date)
+        event = None
+        if self.previous_event:
+            previous_event = self.previous_event
+        else:
+            previous_event = (
+                Event.objects
+                .filter(city=self.city, country=self.country)
+                .order_by("-date")
+                .first()
+            )
+
```

Comment:
@aniav I think this logic here is wrong, as I discussed with Ola on this pull request earlier. I think we shouldn't be treating "previous_event" as event to copy, as the question asked in the form is "have you organized an event before? if so, choose which". If someone organized an event in Berlin, but then they moved to London, they'd answer this question with "Berlin", but that doesn't mean we should copy the Berlin event with a new date, but we should copy last London event.